addons/website_event_questions/models/event_question.py

```diff
@@ -22,6 +22,7 @@ class EventQuestion(models.Model):
     once_per_order = fields.Boolean('Ask only once per order',
                                     help="If True, this question will be asked only once and its value will be propagated to every attendees."
                                          "If not it will be asked for every attendee of a reservation.")
+    mandatory_answer = fields.Boolean('Mandatory Answer', default=False)
```

Comment:
False is already the default values for boolean fields. No need to restate it :)

Reply:
Thanks, indeed, I have changed it